src/PanoViewer/PanoViewer.js

```diff
@@ -112,12 +124,47 @@ export default class PanoViewer extends Component {
 	 */
 
 	/**
+		* Getting the video information that the viewer is currently using.
+		* @ko 뷰어가 현재 사용 중인 비디오 정보를 얻습니다.
+		* @method eg.view360.PanoViewer#getVideo
+		* @return {VideoInfo}
+		*/
+	getVideo() {
+		if (!this._isVideo) {
+			return null;
+		}
+
+		return {
+			video: this._image,
+			type: this._imageType
+		};
```

Comment:
Be careful its return type is differ from `getImage()`;

getVideo returns,
  - video: <video> element
  - **type**: projectionType (equirectangular or  cubemap)

getImage returns
  - image: Image Object
  - **imageType**: projectionType (equirectangular or  cubemap)

Both return projection type but it expose other interface(type/imageType) each other.

Reply:
@happyhj As you mentioned in offline, I think it would be better not to expose type information.

Reply:
I found imageType accessing code that use like below.

```js
panoViewer.setImage({
  image: image,
  imageType: panoViewer.getImage().imageType
});
```

We need to decide name of imageType getter.

  1. projectionType
  2. imageType
  3. type
  4. and others?